### ISSUE TITLE
ArbReal printing incorrect exponent

### ISSUE THREAD

**Issue by jackschmidt** [JeffreySarnoff/ArbNumerics.jl#17]
1e-10 is printed as 1e-1, 1e-20 as 1e-2, etc.

```julia
julia> import ArbNumerics

julia> ArbNumerics.ArbReal("1e-10")
1.000000000000000000000000000000e-1

julia> ArbNumerics.ArbReal("1e-10")+1
1.0000000001

julia> Pkg.status("ArbNumerics")
    Status `~/.julia/environments/v1.1/Project.toml`
  [7e558dbc] ArbNumerics v0.5.3

julia> VERSION
v"1.1.1"

```

**Comment by JeffreySarnoff** [comment]
thank you

**Comment by JeffreySarnoff** [comment]
Very much appreciate your isssue -- it is fixed in master, merge of version v0.5.4 is pending.
